src/Metadata/Resource/ResourceMetadataCollection.php

```diff
@@ -61,6 +61,10 @@ public function getOperation(?string $operationName = null, bool $forceCollectio
                 if ($name === $operationName) {
                     return $this->operationCache[$operationName] = $operation;
                 }
+
+                if ($operation->getUriTemplate() === $operationName) {
```

Comment:
```suggestion
               if ($operationName === $operation->getUriTemplate()) {
```
Also, the other `if` instances in this loop don't respect the Yoda Style.